packages/amplify_api/example/integration_test/graphql_tests.dart

```diff
@@ -285,6 +285,23 @@ void main() {
         expect(data?.id, equals(blog.id));
       });
 
+      testWidgets('should CREATE a blog with Model helper and embedded type',
```

Comment:
nit: I think we should use custom type for consistency... non-model, embedded different names could be confusing :D custom type is the formal name referred by Admin UI

Reply:
ok, will update, good call